fe/fe-core/src/test/java/com/starrocks/analysis/SetStmtTest.java

```diff
@@ -211,4 +211,51 @@ public void testMaterializedViewRewriteMode() throws AnalysisException {
             }
         }
     }
+
+    @Test
+    public void testFollowerQueryForwardMode() throws AnalysisException {
+        // normal
```

Comment:
need to cover the logic that after session var is set to true, query will be forwarded to leader

Reply:
added it.